kytos/core/config.py

```diff
@@ -9,7 +9,7 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 from configparser import ConfigParser
 
-from kytos.core import __version__
+from kytos.core.metadata import __version__
```

Comment:
I've implemented with _metadata, to avoid _metadata being always exposed to the kytos users (it is not necessary).